iothub/device/tests/Pipeline/RetryDelegatingHandlerTests.cs

```diff
@@ -444,6 +446,7 @@ public bool ShouldRetry(int currentRetryCount, Exception lastException, out Time
             {
                 Counter++;
                 lastException.Should().BeOfType(typeof(IotHubClientException));
+                ((IotHubClientException)lastException).StatusCode.Should().Be(IotHubStatusCode.NetworkErrors);
```

Comment:
Same question here

Reply:
No, similar answer as above. `NetworkErrors` is expected.